docs/reference/cluster/stats.asciidoc

```diff
@@ -188,8 +193,19 @@ Will return, for example:
           "classname": "org.elasticsearch.ingest.useragent.IngestUserAgentPlugin",
           "has_native_controller": false
         }
-      ]
+      ],
+      "network_types" : {
+         "transport_types" : {
+            "netty4" : 1
+         },
+         "http_types" : {
+            "netty4" : 1
+         }
+      }
    }
 }
 --------------------------------------------------
+// TESTRESPONSE[s/"plugins": \[[^\]]*\]/"plugins": $body.$_path/]
+// TESTRESPONSE[s/: (\-)?[0-9]+/: $body.$_path/]
+// TESTRESPONSE[s/: "[^"]*"/: $body.$_path/]
```

Comment:
Also, I wasn't able to test this one on Windows and I suspect it fails due to the open file descriptor field.